test/test.js

```diff
@@ -78,10 +78,10 @@ describe('ConnectionPool', function () {
     it('min', function (done) {
         this.timeout(timeout);
 
-        var poolConfig = {min: 2};
+        var poolConfig = { min: 2 };
```

Comment:
@jschell12 I think these other changes might be making this harder to merge (it would be for me anyway, if I was the one who had to merge it)